### PR TITLE
Part2 Issue 1569 - Install script for installing the agent-in-k8s container onto the edge cluster

### PR DESCRIPTION
**Pull request by LiilyZhang**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,4 +23,5 @@ minikube-setup-by-test
 /i18n_messages/catalog.go
 /cli/i18n_messages/catalog.go
 go.sum
-/anax-in-k8s/bin/*
+/anax-in-k8s/hzn
```
Comment (dabooz) [inline .gitignore]: ./anax/Makefile clean target should remove these 2 files

File: Makefile [hunk 2]
```diff
@@ -382,7 +386,22 @@ anax-k8s-promote:
 	docker tag $(ANAX_K8S_UBI_IMAGE_STG) $(ANAX_K8S_UBI_IMAGE_PROD)
 	docker push $(ANAX_K8S_UBI_IMAGE_PROD)
 
-anax-k8s-package: anax-k8s-image anax-k8s-push-only
+anax-k8s-package: anax-k8s-image
+	@echo "Packaging anax-k8s container"
+	if [[ $(shell tools/image-exists $(FSS_REGISTRY) $(ANAX_K8S_IMAGE_BASE) $(ANAX_K8S_IMAGE_VERSION) 2> /dev/null) == "0" || $(FSS_OVERRIDE) != "" ]]; then \
+		echo "Pushing ESS docker image $(ANAX_K8S_IMAGE_BASE):$(ANAX_K8S_IMAGE_VERSION)"; \
```
Comment (dabooz) [inline Makefile]: Fix all the echo statements in this target

File: Makefile [hunk 3]
```diff
@@ -382,7 +386,22 @@ anax-k8s-promote:
 	docker tag $(ANAX_K8S_UBI_IMAGE_STG) $(ANAX_K8S_UBI_IMAGE_PROD)
 	docker push $(ANAX_K8S_UBI_IMAGE_PROD)
 
-anax-k8s-package: anax-k8s-image anax-k8s-push-only
+anax-k8s-package: anax-k8s-image
+	@echo "Packaging anax-k8s container"
+	if [[ $(shell tools/image-exists $(FSS_REGISTRY) $(ANAX_K8S_IMAGE_BASE) $(ANAX_K8S_IMAGE_VERSION) 2> /dev/null) == "0" || $(FSS_OVERRIDE) != "" ]]; then \
```
Comment (dabooz) [inline Makefile]: Fix all the FSS_REGISTRY references, I think should be DOCKER_IMAGE_REPO?

Comment (dabooz) [inline Makefile]: FSS_OVERRIDE also seems out of place, change it to ANAX_K8S_OVERRIDE

File: anax-in-k8s/Dockerfile [hunk 4]
```diff
@@ -20,8 +20,10 @@ WORKDIR /home/anaxuser
 COPY script/* /home/anaxuser/
 COPY config/* /etc/horizon/
 
-ADD bin/* /usr/horizon/bin/ 
-ADD bin/* /usr/bin/
+ADD anax /usr/horizon/bin/
+ADD hzn /usr/horizon/bin/ 
+ADD anax /usr/bin/
```
Comment (dabooz) [inline anax-in-k8s/Dockerfile]: What is the reason for 2 copies of the binaries?

File: Makefile [hunk 5]
```diff
@@ -358,11 +360,13 @@ promote-agbot:
 promote-mac-pkg-and-docker: promote-mac-pkg promote-docker promote-agbot
 
 anax-k8s-image:
-	rm -rf $(ANAX_K8S_CONTAINER_DIR)/bin/*
-	cp $(EXECUTABLE) $(ANAX_K8S_CONTAINER_DIR)/bin 
-	cp $(CLI_EXECUTABLE) $(ANAX_K8S_CONTAINER_DIR)/bin
-	@echo "Producing ANAX K8S docker image $(ANAX_K8S_IMAGE_STG)"
+	rm -rf $(ANAX_K8S_CONTAINER_DIR)/anax
```
Comment (dabooz) [inline Makefile]: Remove the anax-k8s-push-only and anax-k8s-promote targets because they wont be needed.

File: anax-in-k8s/Dockerfile (hunk 4 above)
Comment (LiilyZhang) [reply anax-in-k8s/Dockerfile]: anax.service will need them from /usr/horizon/bin/, and hzn register will need them from /usr/bin/
I will try add them to /usr/horizon/bin/ and update PATH

Comment (LiilyZhang) [reply anax-in-k8s/Dockerfile]: Oh, maybe I can just add anax to /usr/horizon/bin/ and add hzn to /usr/bin/

**Pull request merged**